HW03/Ivan_Papushoi/app.rb

```diff
@@ -0,0 +1,65 @@
+# The core idea of the app:
+# there are objects "tasks" that assign to students and they should solve it
+# mentor see results, check it, and set statuses 'success' and 'denied'
+# if it was denied, students try to resolve it and sends results again.
+# Mentors and students can requests tasks selections
+# which were created, passed, solved and denied
+
+require_relative 'student'
+require_relative 'mentor'
+require_relative 'task'
+
+students = []
+students << student1 = Student.new('Igor')
+students << student2 = Student.new('Sergey')
```

Comment:
Would be better to go separately 
```suggestion
students = []
student1 = Student.new('Igor')
student2 = Student.new('Sergey')
students << student1
students << student2
```

Reply:
Done